Tasks/PublishCodeCoverageResultsV1/package.json

```diff
@@ -1,6 +1,11 @@
 {
   "dependencies": {
-    "azure-pipelines-task-lib": "2.8.0",
-    "azure-pipelines-tasks-codecoverage-tools": "1.0.0"
+    "@types/node": "^10.17.0",
+    "@types/mocha": "^5.2.7",
+    "azure-pipelines-task-lib": "3.0.6-preview.1",
```

Comment:
Could it make sense to specify it as '^3.0.6-preview.1'?

Reply:
Thanks! Fixed it